shared/profile/user/container.tsx

```diff
@@ -54,6 +54,11 @@ const connected = Container.namedConnect(
       const {followersCount, followingCount, followers, following, reason, webOfTrustEntries} = d
       const mutualFollow = followThem && Constants.followsYou(state, username)
 
+      const filteredWot = (webOfTrustEntries ?? []).filter(Constants.showableWotEntry)
+      const hasAlreadyVouched = filteredWot.some(entry => entry.attestingUser === myName)
```

Comment:
i moved these down to mergeprops per jacob's suggestion because it thrashes up here. regardless, i have a PR that dramatically simplifies all this by moving to hooks from the separate-container approach: https://github.com/keybase/client/pull/23589/commits/90c25bab085d4c9912c601774b1d7e2e586c4c05